nilearn/image/tests/test_resampling.py

```diff
@@ -92,14 +92,14 @@ def test_identity_resample():
     # Test with non native endian data
 
     # Test with big endian data ('>f8')
-    for interpolation in ['nearest', 'continuous']:
+    for interpolation in ['nearest', 'linear','continuous']:
```

Comment:
Sorry to always come back with a new comment, but the Python style guide (known as 'PEP8') says that there should be a space here, after the coma: always a space after, never before.

Reply:
No worries, got to learn these things at some point :-)